### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by bernardnormier**

### CODE REVIEW

File: src/IceRpc.Slice/README.md [hunk 1]
```diff
@@ -56,9 +59,10 @@ server.Listen();
 await CancelKeyPressed;
 await server.ShutdownAsync();
 
-// Class IceRpc.Slice.Service is an IceRPC dispatcher. This dispatch is implemented by
-// Chatbot and helper code provided by the IGreeterService interface generated by the
-// Slice compiler.
+// IGreeterService is an interface generated by the Slice compiler.
+// The [SliceService] attribute instructs the Slice Service source generator (provided by
+// IceRpc.Slice.Generators.dll) to implement IDispatcher by directing "greet" requests to
```
Comment (pepone) [inline src/IceRpc.Slice/README.md:64]:
```suggestion
// IceRpc.Slice.Generators.dll) to implement IDispatcher by directing "Greet" requests to
```

Like in the operation name

**Review by pepone — APPROVED** [review]

File: src/IceRpc.Slice/README.md (hunk 1 above)
Comment (bernardnormier) [reply src/IceRpc.Slice/README.md:64]: The operation name is greet in lower-case.

File: README.md [hunk 2]
```diff
@@ -120,10 +120,10 @@ And you can use IceRPC with a [DI container][icerpc-with-di]—or not. It's all
 ## Choose your IDL
 
 IceRPC provides a first-class byte-oriented API that allows you to make RPCs with the [IDL] and serialization format of
-your choice. For example, you can easily send Protobuf messages over IceRPC as illustrated by the [GreeterProtobuf]
-example.
+your choice.
 
-Another option—and the most common choice—is to use Slice to define the contract between your clients and servers.
+IceRPC comes with full support for two IDLs: Slice (described below) and [Protobuf]. You can select either Slice or
```
Comment (InsertCreativityHere) [inline README.md:125]: You _use_ an IDL to define contracts, you don't _select_ an IDL to define contracts.
```suggestion
IceRPC comes with full support for two IDLs: Slice (described below) and [Protobuf]. You can use either Slice or
```

Or you could keep `select` and say:
> You can select either Slice or Protobuf for defining the contract ...

File: README.md [hunk 3]
```diff
@@ -183,6 +183,14 @@ You can define new types with `struct`, `enum`, and `custom`. And you can constr
 [custom] allows you to send any C# type you wish through Slice, in keeping with IceRPC's mantra of modularity and
 extensibility. You just need to provide methods to encode and decode instances of your custom type.
 
+## Protobuf
+
+Protocol Buffers or Protobuf for short is a popular IDL and serialization format developed by Google. It's the preferred
```
Comment (InsertCreativityHere) [inline README.md:188]: Should be some kind of separator for this parenthetical statement.
For some unexplainable reason, commas feel most correct to me.
```suggestion
Protocol Buffers, or Protobuf for short, is a popular IDL and serialization format developed by Google. It's the preferred
```

File: README.md [hunk 4]
```diff
@@ -207,8 +215,8 @@ in-memory transport for testing). Future releases may add additional transports.
 [custom]: https://docs.icerpc.dev/slice2/language-guide/custom-types
 [Documentation]: https://docs.icerpc.dev
 [Examples]: examples
-[Getting started]: https://docs.icerpc.dev/getting-started
-[GreeterProtobuf]: examples/GreeterProtobuf
+[Getting started]: https://docs.icerpc.dev/getting-
```
Comment (InsertCreativityHere) [inline README.md:218]: Pretty sure this link got broken:
```suggestion
[Getting started]: https://docs.icerpc.dev/getting-started
```

File: tools/IceRpc.Protobuf.Tools/README.md [hunk 5]
```diff
@@ -10,27 +10,27 @@ generator is a .NET program and requires .NET 8 or later.
...
 The `protoc-gen-icerpc-csharp` generator generates a `.IceRpc.cs` file for each `.proto` file. The generated C# file
-has the same name as the corresponding `.proto` file, but with the name converted to camel case. For example,
-`greeter.proto` is compiled into `Greeter.cs`.
+has the same name as the corresponding `.proto` file, but with the name converted to Pascal case. For example,
+`greeter.proto` is compiled into `Greeter.IceRpc.cs`.
 
 [Source code][source] | [Package][package]
 
-## Protobuf files and Protobuf directories
+## Protobuf files and Protobuf import path
```
Comment (InsertCreativityHere) [inline tools/IceRpc.Protobuf.Tools/README.md:28]: Should this be plural? `import paths`?
Maybe it's correct as-is, I can't tell from the context. But `directories` used to be plural.

File: tools/IceRpc.Protobuf.Tools/README.md [hunk 6]
```diff
@@ -54,7 +54,8 @@ or any of its subdirectories.
 The Protobuf import path is an aggregate of the `ProtoImportPath` defined in your project (if any) and the
 `ProtoImportPath` defined in NuGet packages referenced by your project.
 
-For example, if your project's Protobuf files reference definitions in directory `common/proto`:
+For example, if your project's Protobuf files import files in directory `common/proto`, set your `ProtoImportPath` as
+follows:
```
Comment (InsertCreativityHere) [inline tools/IceRpc.Protobuf.Tools/README.md:58]: I'm not sure what this is trying to say.

**Review by InsertCreativityHere — APPROVED** [review]
Looks good!
It's nice to see the parity between Slice and Protobuf.

File: tools/IceRpc.Protobuf.Tools/README.md (hunk 5 above)
Comment (bernardnormier) [reply tools/IceRpc.Protobuf.Tools/README.md:28]: Path = several directories.

**Pull request merged**